noir-projects/aztec-nr/aztec/src/context/private_context.nr

```diff
@@ -550,7 +550,9 @@ impl PrivateContext {
         };
     }
 
-    fn next_counter(&mut self) -> u32 {
+    // This function got exposed publicly to be able to access it in ContractClassRegistry.
+    // TODO: Should we introduce an abstraction such that accessing this externally is not needed?
```

Comment:
I think unblocking the next noir sync is more important than this so just lazily added here and below a TODO